oss_local_scripts/make_egg.sh

```diff
@@ -103,6 +103,9 @@ build_source() {
   echo -e "\n\n\n================= Build ${BUILD_NUMBER} ================\n\n\n"
   # Configure
   cd ${WORKSPACE}
+  # Clean up, except for the deps.
+  rm -rf "${WORKSPACE}/debug" "${WORKSPACE}/release"
```

Comment:
Eh... Lets not do this. In almost all circumstances this is unnecessary. And it slows down the build quite significantly since now it *really* has to rebuild everything.